.github/labeler.yml

```diff
@@ -14,11 +15,37 @@ area/integrations:
   - .github/**/*
   - .travis/**/*
 area/testing/jepsen:
-  - contrib/jepsen/**/* 
+  - contrib/jepsen/**/*
 area/commercial:
   - ee/**/*
-  - wiki/content/enterprise-features/**/*
+area/enterprise:
```

Comment:
why do we have two tags for this code, area/enterprise and area/commercial?

Reply:
The two were already there when I arrived. So I used both. Which should we use? At the time, no one clarified their use, so I adopted both.

Reply:
See 
https://github.com/dgraph-io/dgraph/labels/area%2Fcommercial
https://github.com/dgraph-io/dgraph/labels/area%2Fenterprise

Enterprise looks more used.

Reply:
let's use that then.